### DISCUSSION TITLE
Add Salesforce diagrams icons to library

### DISCUSSION THREAD

**Discussion by ernestovalent** [jgraph/drawio#3169] [Ideas]
**Is your feature request related to a problem?**
No, I would like icons to be added

**Describe the solution you'd like**
Now Salesforce have a design guide to make diagrams using design Templates (as well as [lucidchart](https://www.lucidchart.com/blog/salesforce-shape-library)).

**Describe alternatives you've considered**
Lucidchart have integration icons [here](https://www.lucidchart.com/blog/salesforce-shape-library).
Example
![image](https://user-images.githubusercontent.com/65668395/166837265-346dda8f-8474-4ebd-a086-8cf1c836db57.png)

**Additional context**
About Salesforce Templates:

- Videos: [Youtube playlist](https://www.youtube.com/playlist?list=PLn15mOuXqGJUOc-iPmMcLb0HGi4wusJpv)
- Docs: [architect](https://architect.salesforce.com/design/diagram-templates/doc-implementation-templates/)

**Comment by davidjgraph** [comment]
@egrimate
